src/System.Net.Mail/tests/Functional/AttachmentTest.cs

```diff
@@ -32,6 +32,23 @@ public void ConstructorNullName()
             Assert.Equal(null, attach.Name);
         }
 
+        [Fact]
+        public void ConstructorPathName()
+        {
+            string attachFile = Path.GetTempFileName();
+            try
+            {
+                var fileStream = File.Create(attachFile);
+                fileStream.Close();
+                Attachment attach = new Attachment(attachFile);
```

Comment:
You can change "Attachment" to "var" since it is obvious what the type is.

Reply:
The other tests in this file use `Attachment` so I'll keep it to be consistent